### PR TITLE
New version: THS.THS.Hevo version 10.1.1.1

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#8292](https://github.com/SpecterShell/Dumplings/actions/runs/10993957022).

<details>

<summary>Log</summary>

````
Updated: 10.0.1.1 → 10.1.1.1
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/174742)

### CODE REVIEW

File: manifests/t/THS/THS/Hevo/10.1.1.1/THS.THS.Hevo.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ShortDescription: Brand new stock quotes software by Royalflush
+Description: THS Hevo is a dedicated work based on big data and AI that uses the newest generation of communication protocols to deliver accurate stock quotes in milliseconds and can be customized according to investor's investment style. Not only is it equipped with a unique indicator platform matrix that can be redefined with Python to discover the movement of major funds, but also exclusive develop special indicators such as TD Sequential, bull-bear line and moving averages to accurately grasp the buying and selling points and significantly improve the winning percentage of picking tops and bottoms. It is a heavyweight product of Royalflush that fulfills the mission of "Making Investment Easier".
+# Moniker:
+Tags:
+- finance
+- index
+- market
+- stock
+# ReleaseNotes:
+ReleaseNotesUrl: https://t.10jqka.com.cn/pid_380472825.shtml
```
Comment (stephengillie) [inline manifests/t/THS/THS/Hevo/10.1.1.1/THS.THS.Hevo.locale.en-US.yaml:27]:
```suggestion
# ReleaseNotesUrl: 
```

**Review by stephengillie — DISMISSED** [review]
> Url validation error
> 
> * manifests\t\THS\THS\Hevo\10.1.1.1
>   
>   * https://t.10jqka.com.cn/pid_380472825.shtml
>     
>     * Http status code: Forbidden

This page works for me, but our data center appears to be blocked.

File: manifests/t/THS/THS/Hevo/10.1.1.1/THS.THS.Hevo.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- 股市
+- 股票
+- 财经
+- 金融
+ReleaseNotes: |-
+  1.【新增】管理标记综合页及标记优化
+  2.【新增】决策页面
+  3.【新增】行业风格
+  4.【优化】盘口关键数据
+ReleaseNotesUrl: https://t.10jqka.com.cn/pid_380472825.shtml
```
Comment (spectopo) [inline manifests/t/THS/THS/Hevo/10.1.1.1/THS.THS.Hevo.locale.zh-CN.yaml:33]:
```suggestion
# ReleaseNotesUrl: https://t.10jqka.com.cn/pid_380472825.shtml
```

Comment (SpecterShell) [inline manifests/t/THS/THS/Hevo/10.1.1.1/THS.THS.Hevo.locale.zh-CN.yaml:33]:
```suggestion
# ReleaseNotesUrl: https://t.10jqka.com.cn/pid_380472825.shtml
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**